src/helpers.js

```diff
@@ -0,0 +1,33 @@
+import { defineCustomElement as defineCE } from '../esm/es5/corporate-ui.core'
```

Comment:
As we are now exporting "defineCustomElements", the as syntax is no longer needed

Reply:
That's correct...will change it and add a demo in Storybook